instrumentation/que/lib/opentelemetry/instrumentation/que/patches/que_job.rb

```diff
@@ -46,7 +46,7 @@ def enqueue(*args, job_options: {}, **arg_opts)
                 # In Que version 2.1.0 `bulk_enqueue` was introduced and in order
                 # for it to work, we must pass `job_options` to `bulk_enqueue` instead of enqueue.
                 if gem_version >= Gem::Version.new('2.1.0') && Thread.current[:que_jobs_to_bulk_insert]
-                  Thread.current[:que_jobs_to_bulk_insert][:job_options] = job_options.merge(tags: tags)
+                  Thread.current[:que_jobs_to_bulk_insert][:job_options] = Thread.current[:que_jobs_to_bulk_insert][:job_options].merge(tags: tags)
```

Comment:
Please add an automated test to ensure we do not introduce regressions in the future.

Reply:
There already was a test for this behaviour but the test itself had an issue, where `job_options` were passed to `.enqueue` not `.job_enqueue`. Fixed in f4d5c4b

https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/f4d5c4b3f190b354f99dd41ff65db486beec4351/instrumentation/que/test/opentelemetry/instrumentation/que_test.rb#L413C13-L415C16